### PR TITLE
Added new 2 Doctors on Doctors Page

### PR DESCRIPTION
**Pull request by Devmangrani**
I have Added 2 More Doctors Information on Doctors Page

### CODE REVIEW

**Comment by yazdanhaider** [comment]
@Devmangrani change the description of the doctors you have added.

File: Frontend/src/App.jsx [hunk 1]
```diff
@@ -26,7 +26,9 @@ function App() {
...
     { id: 6, name: "Dr. Alex", specialty: "Gynecologist", patients: 350, appointments: 650, experience: 10, qualifications: "MD, PhD", image: "https://randomuser.me/api/portraits/men/94.jpg", bio: "Dr. Alex is an experienced gynecologist providing comprehensive women's health care, specializing in reproductive health, pregnancy care, and preventive medicine for women of all ages."},
-    { id: 7, name: "Dr. Wilson", specialty: "General Medicine", patients: 250, appointments: 700, experience: 20, qualifications: "MD, FACP", image: "https://randomuser.me/api/portraits/women/6.jpg", bio: "Dr. Wilson is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
+    { id: 7, name: "Dr. Wilson", specialty: "General practitioner", patients: 250, appointments: 620, experience: 12, qualifications: "MD, PHD", image: "https://randomuser.me/api/portraits/women/6.jpg", bio: "Dr. Wilson is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
```
Comment (yazdanhaider) [inline Frontend/src/App.jsx:29]: Please update the descriptions for the doctors you added, as all of them currently have the same description.

**Review by yazdanhaider — CHANGES_REQUESTED** [review]
Change the doctor description.

File: Frontend/src/App.jsx (hunk 1 above)
Comment (Devmangrani) [reply Frontend/src/App.jsx:29]: okk i will do it..

**Pull request merged**